### DISCUSSION TITLE
Med Admin

### DISCUSSION THREAD

**Discussion by smiley26** [marsupialproject/marsupial-emr#6] [General]
Hi, 
This EMR looks like it could work well in our sim lab. One quick question that I am unclear about. During the demo when you show med admin you continually say when you scan the med. While I know that's the standard, we do not currently have scanners for our learners. Do you have to scan the med in order to document that it's been administered or is there a workaround... (for example "Scanner Broken")

Thanks. 

Sam

**Comment by marsupialproject** [comment]
Hi there. The only current workaround would be to type the barcode into the scanning box. You would have to do this for the patient's barcode and then any medications as well. I was going to add the "Scanner Broken" functionality but it would be a lot of work and the program was made specifically for barcode scanning. This may come in the future though.

**Comment by smiley26** [comment]
Thanks for clarifying. Does any barcode scanner work and do you have any
recommendations that will work well with your EMR? We are not a hospital
based program but instead at a higher ed institution so do not have a need
for scanners.

**Reply by marsupialproject** [reply]
There's info on barcode scanners and some recommendations in the wiki.
https://github.com/marsupialproject/marsupial-emr/wiki/Marsupial-EMR#barcode-scanner

I'm currently using a mix of the Zebra DS6708, DS2208, and DS2278. The DS2208 is a wired scanner and the DS2278 is wireless, both currently available new. The DS6708 is discontinued but can be found on eBay for cheap. 

FYI If you want to scan QR codes you need a 2D scanner. The 1D scanners are for the typical UPC barcodes and are usually a little cheaper. The Zebra ones I listed are 2D scanners. They also make 1D scanners like the LS2208 which may be cheaper if cost is an issue.
